sdk/servicebus/service-bus/test/deferredMessage.spec.ts

```diff
@@ -120,19 +120,28 @@ async function deferMessage(testMessages: SendableMessageInfo): Promise<ServiceB
   const sequenceNumber = receivedMsgs[0].sequenceNumber;
   await receivedMsgs[0].defer();
 
-  const deferredMsgs = await receiver.receiveDeferredMessage(sequenceNumber);
-  if (!deferredMsgs) {
+  let deferredMsg: ServiceBusMessage | undefined;
+
+  // Randomly choose receiveDeferredMessage/receiveDeferredMessages as the latter is expected to
+  // convert single input to array and then use it
```

Comment:
Can we test with both rather than either, randomly? A bug in one or the other might be hidden with the random approach.

Reply:
Updated to not depend on Math.random() to get the test coverage. See https://github.com/Azure/azure-sdk-for-js/pull/2234/commits/b3195e2d077ca8cbcb870cae73b6a5a05a3120ac